addons/web_tour/static/src/js/tour_manager.js

```diff
@@ -433,7 +433,8 @@ return core.Class.extend(mixins.EventDispatcherMixin, ServicesMixin, {
                 message = _t('<strong><b>Good job!</b> You went through all steps of this tour.</strong>');
             }
             const fadeout = this.tours[tour_name].fadeout;
-            new RainbowMan({message, fadeout}).appendTo(this.$body);
+            var event = new CustomEvent('show-effect', { bubbles: true, detail: {message: message, fadeout: fadeout} });
+            this.$body[0].dispatchEvent(event)
```

Comment:
@aab-odoo I'm not a fan of this. 
But I don't see another way as we do not have the env for the services, or some kind of trigger up method.